src/runner/runESLint.js

```diff
@@ -73,6 +73,7 @@ const mkTestResults = ({
       start: startTime,
       end: endTime,
       duration: endTime - startTime,
+      runtime: endTime - startTime,
```

Comment:
both of these seems silly to me - I'd assume Jest could figure it out on its own... I'll dig into this a bit more tomorrow and attempt to figure out why this would be needed

Reply:
Fix in Jest is https://github.com/jestjs/jest/pull/14473, but that would only fix it for Jest 29 - this regressed in Jest 26